app/src/main/kotlin/com/simplemobiletools/voicerecorder/adapters/ViewPagerAdapter.kt

```diff
@@ -53,7 +53,7 @@ class ViewPagerAdapter(private val activity: SimpleActivity, val showRecycleBin:
     }
 
     fun finishActMode() {
-      (mFragments[1] as? PlayerFragment)?.finishActMode()
+        (mFragments[1] as? PlayerFragment)?.finishActMode()
```

Comment:
why?

Reply:
Formatting did this. Looks like this was not properly formatted originally

Reply:
right, my bad